packages/components/src/Lettera.purs

```diff
@@ -76,15 +80,57 @@ letteraSearchUrl = letteraBaseUrl <> "/list/search"
 letteraAdvertorialUrl :: String
 letteraAdvertorialUrl = letteraBaseUrl <> "/list/active-advertorial"
 
+encodeRequestHeaders :: Array AX.RequestHeader -> Maybe Json
```

Comment:
`Json` should only be used for serialization/deserialization, not this.

Reply:
Is `Show` better than `Json` for displaying formatted errors? Or how about using some custom `formatErrorMessage` function so that we wouldn't be using standard classes for a non-standard purpose?

Reply:
I think that using JSON instead of more human readable ad hoc error messages might make it easier to search for errors in our cloug log system.